### DISCUSSION TITLE
the program does not parse in Russian

### DISCUSSION THREAD

**Discussion by SiVitaliy** [SnosMe/awakened-poe-trade#1074] [Q&A] [answered]
the program does not parse in Russian
![image](https://github.com/SnosMe/awakened-poe-trade/assets/83186271/56d113bb-92a3-4dc2-bf54-8a5c8e3905c0)

![image](https://github.com/SnosMe/awakened-poe-trade/assets/83186271/6cda4320-5665-4518-ba27-6325bb81a46c)

**Comment by SnosMe** [comment, marked as answer]
It does, select it in settings
